ts/receiver/opengroup.ts

```diff
@@ -52,6 +53,14 @@ const handleOpenGroupMessage = async (
 
   const decodedContent = SignalService.Content.decode(dataUint);
 
+  if (!shouldProcessContentMessage({ timestamp: sentTimestamp }, decodedContent, true)) {
+    window?.log?.info(
+      'sogs message: shouldProcessContentMessage is false for message sentAt:',
```

Comment:
related to https://github.com/session-foundation/session-desktop/pull/69#discussion_r1953781456